src/core/Notification/Notification.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { Component, forwardRef, ReactNode } from 'react';
+
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import {
+  HtmlDiv,
+  HtmlDivWithRef,
+  HtmlButton,
+  HtmlButtonProps,
+  HtmlDivWithRefProps,
+} from '../../reset';
+import { hLevels } from '../../reset/HtmlH/HtmlH';
+import { Icon } from '../Icon/Icon';
+import { Heading } from '../Heading/Heading';
+
+import { AutoId } from '../utils/AutoId/AutoId';
+import { getConditionalAriaProp } from '../../utils/aria';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { baseStyles } from './Notification.baseStyles';
+
+export const baseClassName = 'fi-notification';
+
+export const notificationClassNames = {
+  styleWrapper: `${baseClassName}_style-wrapper`,
+  content: `${baseClassName}_content`,
+  label: `${baseClassName}_label`,
+  textContentWrapper: `${baseClassName}_text-content-wrapper`,
+  icon: `${baseClassName}_icon`,
+  iconWrapper: `${baseClassName}_icon-wrapper`,
+  closeButton: `${baseClassName}_close-button`,
+  closeButtonWrapper: `${baseClassName}_close-button-wrapper`,
+  smallScreen: `${baseClassName}--small-screen`,
+  actionElementWrapper: `${baseClassName}_action-element-wrapper`,
+};
+
+export interface NotificationProps extends HtmlDivWithRefProps {
+  /** Style variant. Affects color and icon used.
+   * @default 'neutral'
+   */
+  status?: 'neutral' | 'success' | 'error';
+  /** Set aria-live mode for the Notification text content and label.
+   * @default 'polite'
+   */
+  ariaLiveMode?: 'polite' | 'assertive' | 'off';
+  /** Label for the Notification */
+  labelText?: string;
+  /** Main content of the Notification */
+  children?: ReactNode;
+  /** Elements to be rendered under the notification text. Can be e.g. buttons, links etc. */
+  actionElements?: ReactNode;
+  /** Header variant for Notification */
+  headingVariant?: hLevels;
+  /** Text to label the close button. Visible + `aria-label` in regular size and only used as `aria-label` in small screen variant */
+  closeText: string;
+  /** Click handler for the close button */
+  onCloseButtonClick?: () => void;
+  /** Custom props passed to the close button */
+  closeButtonProps?: Omit<HtmlButtonProps, 'onClick' | 'aria-label'>;
+  /** Use small screen styling */
+  smallScreen?: boolean;
+  /** LabelText is header of component */
+}
```

Comment:
Are we missing `labelText` prop or are we having unwanted JSDoc line for removed prop?

Reply:
Fixed. Removed comment.

Reply:
When did we decide to drop labelText from the api? Also, it's still part of the component below as well as the examples. Are you sure the prop is not needed?

Reply:
It seems that `labelText` is still in the API, but is now moved higher in the interface

https://github.com/vrk-kpa/suomifi-ui-components/blob/3e00e4c720163f7adf36cdd2cfebe40b1847f8fc/src/core/Notification/Notification.tsx#L44-L49